src/plugins/elasticsearch/index.js

```diff
@@ -28,8 +28,8 @@ module.exports = function ({ Plugin }) {
           cert: string(),
           key: string()
         }).default(),
-        apiVersion: string().default('2.0'),
-        engineVersion: string().valid('^2.1.0').default('^2.1.0')
+        apiVersion: Joi.string().default('master'),
```

Comment:
We'd replace master with a specific version whenever we created the version branch, which generally happens at feature freeze.